src/pipecat/transports/base_input.py

```diff
@@ -335,13 +332,6 @@ async def process_frame(self, frame: Frame, direction: FrameDirection):
     # Handle interruptions
     #
 
-    async def _handle_bot_interruption(self, frame: BotInterruptionFrame):
-        """Handle bot interruption frames."""
-        logger.debug("Bot interruption")
-        if self.interruptions_allowed:
-            await self._start_interruption()
```

Comment:
This was actually unnecessary before, because the base class `FrameProcessor` already calls `await self._start_interruption()` when it gets a `StartInterruptionFrame`.